tests/queries/0_stateless/02680_datetime64_monotonic_check.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS 02680_datetime64_monotonic_check;
```

Comment:
I run this test on master and got same result

Reply:
did you run with the debug or release build?
the LOGIC_ERROR will cause abort in debug build.

Reply:
Well I ran it in the release build ...